src/Symfony/Component/DependencyInjection/Loader/FileLoader.php

```diff
@@ -52,7 +52,7 @@ public function __construct(ContainerBuilder $container, FileLocatorInterface $l
      *
      * @param bool|string $ignoreErrors Whether errors should be ignored; pass "not_found" to ignore only when the loaded resource is not found
      */
-    public function import($resource, string $type = null, $ignoreErrors = false, string $sourceResource = null, $exclude = null)
+    public function import(mixed $resource, string $type = null, bool | string $ignoreErrors = false, string $sourceResource = null, $exclude = null)
```

Comment:
I've omitted the type declaration for `$exclude` in order to maintain compatibility with Config 5.4. Shall I keep it that way or shall I bump?

Reply:
let's keep compat